### PR TITLE
ddgr: add page

### LINKED ISSUE #11045: page request: ddgr
### Command description

[`ddgr`](https://github.com/jarun/ddgr) is a tool maintained by the same person as the now archived `googler` tool (which does have a `tldr` page). The README describes it as:

> `ddgr` is a cmdline utility to search DuckDuckGo ([html version](https://html.duckduckgo.com/html/)) from the terminal. While [googler](https://github.com/jarun/googler) is extremely popular among cmdline users, in many forums the need of a similar utility for privacy-aware DuckDuckGo came up. [DuckDuckGo Bangs](https://duckduckgo.com/bang) are super-cool too! So here's `ddgr` for you!

`ddgr` is almost a one-to-one clone of `googler` in terms of interface, so this should only require a modification of the `googler` page.

### Command details

v2.1

### OS

- [X] Android
- [X] Linux
- [X] macOS (osx)
- [ ] sunOS
- [X] Windows

### PR DESCRIPTION
**Pull request by aditya-shrivastavv**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Closes #11045.

### CODE REVIEW

File: pages/common/ddgr.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# ddgr
+
+> Commandline utility to search DuckDuckGo (html version) from the terminal.
+> More information: <https://github.com/jarun/ddg>.
+
+- To initiate an interactive mode:
```
Comment (gutjuri) [inline pages/common/ddgr.md:6]:
```suggestion
- Start interactive mode:
```

File: pages/common/ddgr.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`ddgr`
+
+- Search DuckDuckGo for a keyword:
+
+`ddgr {{keyword}}`
+
+- To limit the search results number:
```
Comment (gutjuri) [inline pages/common/ddgr.md:14]:
```suggestion
- Limit the search results number to N:
```

File: pages/common/ddgr.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`ddgr -n {{N}} {{keyword}}`
+
+- To show complete URL in search results:
```
Comment (gutjuri) [inline pages/common/ddgr.md:18]:
```suggestion
- Show the complete URL in search results:
```

File: pages/common/ddgr.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`ddgr -x {{keyword}}`
+
+- Search DuckDuckGo for a keyword and open the first result in a browser:
```
Comment (gutjuri) [inline pages/common/ddgr.md:22]:
```suggestion
- Search DuckDuckGo for a keyword and open the first result in the browser:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thank you very much! I have some suggestions below.

**Comment by aditya-shrivastavv** [comment]
Requested changes applied 👍. I will be more language conscious next time. Thank you

**Review by gutjuri — APPROVED** [review]
Thanks!

**Review by sebastiaanspeck — APPROVED** [review]
LGTM!

**Pull request merged**